java/ql/test/library-tests/typeflow/uniontypeflow.expected

```diff
@@ -0,0 +1,24 @@
+| UnionTypes.java:11:5:11:5 | m | 2 | ConcurrentHashMap<String,String> | false |
+| UnionTypes.java:11:5:11:5 | m | 2 | LinkedHashMap<String,String> | false |
+| UnionTypes.java:26:10:26:10 | x | 2 | A2 | true |
+| UnionTypes.java:26:10:26:10 | x | 2 | A3 | false |
```

Comment:
Why is this not `true`?

Reply:
Because `A3` has no subtypes - in those cases we just report upper bounds and don't bother with the lower bounds.